keyboards/splitkb/zima/keymaps/default/readme.md

```diff
@@ -0,0 +1 @@
+# The default keymap for zima
```

Comment:
Though functionality may be found in code, the readme for the default keymap should contain the keymap for reference.

Reply:
will do